src/components/TransactionDetails/TransactionDetails.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import moment from 'moment';
+
+import { transactionsTotalsSelector } from '../../redux/modules/transactionsList/selectors';
+
+import NavLink from '../NavLinks/NavLink.jsx';
+import TransactionDetailsList from './TransactionDetailsList.jsx';
+
+import { HOME, TRANSACTIONS } from '../../routes';
```

Comment:
OK, here you use this approach (and it's better as for me), but on the main page
```
import * as Routes from '../../routes';
```

Reply:
OK, will use this approach onwards